public/css/widget/view-mode-switcher.less

```diff
@@ -28,6 +28,11 @@
     &:not(:last-of-type) {
       border-right: 1px solid @low-sat-blue-dark;
     }
+
+    i {
+      // fix height for Chrome
+      display: block;
```

Comment:
That's not really valid. The parent is an inline element.

Reply:
Block elements in inline elements aren't valid in HTML for structural reasons, afaik. But I don't see why this shouldn't be valid done with CSS.

Besides that, `inline` and `inline-block` result in crazily inconsistent and incomprehensible value rounding of the rendered height across browsers.

In the end `display: block` is the only thing I found, that works.

Reply:
nvm, seems to be valid inside inline **-block**.